src/fq_poly_templates/mul_reorder.c

```diff
@@ -12,6 +12,11 @@
 
 #ifdef T
 
+/* Inclusion of both mul_reorder.c and sqr_reorder.c can screw things up by
+ * multiple definitions. If changes are made to this file, make sure that
+ * proper changes are made to the other file as well. */
+#define MUL_REORDER_C
```

Comment:
@fredrik-johansson I added these guards to `mul_reorder.c` and `sqr_reorder.c` to guard against multiple definitions of `__vec_init`, `__vec_init2` and so on. Make sure that if anyone alters their content, that proper changes are also made to them (if necessary) to avoid redundant code.